### DISCUSSION TITLE
Welcome to Elan Discussions!

### DISCUSSION THREAD

**Discussion by richardpawson** [elan-language/LanguageAndIDE#624] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EerkeB** [comment]
Hi, how does your Elan relate to https://en.wikipedia.org/wiki/ELAN_(programming_language) ? Me: a former student of the designer of this language, Kees (CHA) Koster, at the then Catholic University of Nijmegen, NL.

**Comment by richardpawson** [comment]
#Hi, how does your Elan relate to https://en.wikipedia.org/wiki/ELAN_(programming_language) ?

Not at all, I'm afraid - except in name (though our usage is camel-case). I was not aware of that ELAN when I came up with the name Elan, though I became aware of it when I searched the name. I know, or know of, a lot of  programming languages, but when our project started I came up with 50 (literally) names, all of which I thought were original and all of which, it turns out were the name of some extant or fomer language! I figured that since a) the ELAN you studied is no longer active and b) it was only ever active in Netherlands and Germany I believe (where our focus is on the UK) that there was little opportunity for confusion, It was, and is, a good name.  There are other (non-language) software tools that use the name also.


Other people have found that Wikipedia page while searching for our Elan project, but you are the first I have heard from that used (I am assuming) that ELAN.

When we get closer to launching (still at Beta, presently) I intend to start a new Wikipedia page, and add disambiguation references both ways.


Thanks for your interest.

**Reply by BernardUK** [reply]
I have pursued this a little by asking for advice from the Wikipedia community in the 'Tea House'. My question and various editors' replies are here: https://en.wikipedia.org/wiki/Wikipedia:Help_desk/Archives/2024_December_13. As you can see, the issue is really only the maturity of this Elan project and the notability criterion which will, we hope, be satisfied when articles get published that are not penned by Elan's esteemed creator. We have to keep in mind that Wikipedia editors do not tolerate what looks like advertising!
